src/test/java/org/candlepin/insights/pinhead/PinheadServiceTest.java

```diff
@@ -84,4 +96,20 @@ public OrgInventory getConsumersForOrg(String orgId, Integer perPage, String off
         assertEquals(consumer3, consumers.get(2));
         assertEquals(consumer4, consumers.get(3));
     }
+
+    @Test
+    public void testPinheadServiceRetry() throws Exception {
+        PinheadApi testApi = Mockito.mock(PinheadApi.class);
+        when(
+            testApi.getConsumersForOrg(anyString(), any(Integer.class), nullable(String.class))
+        ).thenThrow(ApiException.class);
```

Comment:
Can you please add a line to this test such as `retryTemplate.setBackOffPolicy(new NoBackOffPolicy());` (or something with similar effect), so we can avoid adding ~9 seconds to the test?

Reply:
Lol.  Yeah